README.md

```diff
@@ -51,18 +191,19 @@ appTags := application.New("OrderingApp", "inventory")
 
 ### 2. Set Up a Wavefront Sender
 
-A "Wavefront sender" is an object that implements the low-level interface for sending data to Wavefront. You can choose to send data using either the [Wavefront proxy](https://docs.wavefront.com/proxies.html) or [direct ingestion](https://docs.wavefront.com/direct_ingestion.html).
+A `Wavefront sender` object implements the low-level interface for sending data to Wavefront. You can choose to send data using the [Wavefront proxy](https://docs.wavefront.com/proxies.html) or [direct ingestion](https://docs.wavefront.com/direct_ingestion.html).
 
-* If you have already set up a Wavefront sender for another SDK that will run in the same process, use that one. (For details, see [Share a Wavefront Sender](https://github.com/wavefrontHQ/wavefront-sdk-go/blob/master/docs/sender.md#share-a-wavefront-sender).)
+* If you have already set up a Wavefront sender for another SDK that runs in the same process, use that one. (For details, see [Share a Wavefront Sender](https://github.com/wavefrontHQ/wavefront-sdk-go/blob/master/docs/sender.md#share-a-wavefront-sender).)
```

Comment:
We direct the user to the Go SDK docs here.. This might confuse the user (Opentracing SDK to Go SDK). Instead, I can duplicate the content in the OpenTracing Go SDK too?